### PR TITLE
S. Rose new reference topic: delete user by ID

### PR DESCRIPTION
**Pull request by shellyannrose**
Added delete user by id link

Create users-delete-user-by-id.md

Added delete user reference topic

Update users-delete-user-by-id.md

Fixed misspelled word

Update user.md

Fixed link to my new delete ref topic

Update users-delete-user-by-id.md

Removed unneeded text at top of page

Update users-delete-user-by-id.md

Added a related topic section

Update users-delete-user-by-id.md

Fixed link in related topic section

Update users-delete-user-by-id.md

Another link fix

Update users-delete-user-by-id.md

Changed tense of verb

### CODE REVIEW

File: docs/api/users-delete-user-by-id.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+
```
Comment (rbwatson) [inline docs/api/users-delete-user-by-id.md]:
```suggestion
```
Please remove the blank lines before the title.
The title should be the first line of the file, unless there is metadata.

File: docs/api/users-delete-user-by-id.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (rbwatson) [inline docs/api/users-delete-user-by-id.md]:
```suggestion
```

File: docs/api/users-delete-user-by-id.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (rbwatson) [inline docs/api/users-delete-user-by-id.md]:
```suggestion
```

File: docs/api/users-delete-user-by-id.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+```shell
+
+{server_url}/users/{id}
+```
+
+## Params
+
+| Parameter name | Type | Description |
+| -------------- | ------ | ------------ |
+| `id` | number | The record ID of the user to delete|
```
Comment (rbwatson) [inline docs/api/users-delete-user-by-id.md]:
```suggestion
| `id` | number | The record ID of the user to delete |
```
There should be a space between the text and the table delimiters.

File: docs/api/users-delete-user-by-id.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+| Status value | Return status | Description |
+| ------------- | ----------- | ----------- |
+| 200 | Success | Action completed successfully |
+| 202 | Accepted| Action has been queued |
+| 204 | No Content| Action has been performed, but the response does not include an entity |
+| 404 | Error | Specified user record not found |
+|  ECONNREFUSED | N/A | Service is offline. Start the service and try again. |
+
+## Related topics
+[Enroll a new user](../tutorials/enroll-a-new-user)
```
Comment (rbwatson) [inline docs/api/users-delete-user-by-id.md:51]:
```suggestion
[Enroll a new user](../tutorials/enroll-a-new-user)

```
Please add a blank line after the last line of the file.
Thanks!

**Review by rbwatson — CHANGES_REQUESTED** [review]
Thanks for the new topic!
Please fix the minor details I noted in your new file and then it will be ready to merge.

File: docs/api/users-delete-user-by-id.md (hunk 1 above)
Comment (shellyannrose) [reply docs/api/users-delete-user-by-id.md]: Done. Thanks.

File: docs/api/users-delete-user-by-id.md (hunk 2 above)
Comment (shellyannrose) [reply docs/api/users-delete-user-by-id.md]: I'm not sure what the minus means in this context; what should I remove?

File: docs/api/users-delete-user-by-id.md (hunk 4 above)
Comment (shellyannrose) [reply docs/api/users-delete-user-by-id.md]: Done. Thanks.

File: docs/api/users-delete-user-by-id.md (hunk 5 above)
Comment (shellyannrose) [reply docs/api/users-delete-user-by-id.md:51]: Done. Thanks.

**Comment by shellyannrose** [comment]
Hi Bob, I committed the changes you suggested. What else must I do to make this safe to merge? Thanks.

**Review by rbwatson — CHANGES_REQUESTED** [review]
LGTM
Can you squash your commits into just one for the pull request, please?

Thanks!

File: docs/api/users-delete-user-by-id.md (hunk 2 above)
Comment (rbwatson) [reply docs/api/users-delete-user-by-id.md]: There's nothing you need to do. The "-" is part of the difference display and not your code.
It means that in the suggested change, the line is removed.
The red tint also indicates a removal.

**Review by rbwatson — APPROVED** [review]
Rebased and squashed to merge

**Pull request merged**